integration/images/volume-ownership/Dockerfile

```diff
@@ -12,7 +12,8 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-FROM ubuntu
+# Use alpine since busybox doesn't have "nogroup" group.
+FROM ghcr.io/containerd/alpine:3.14.0
```

Comment:
Changing to `alpine` makes sense (to get a smaller image), but see my comment above; I don't think we need to be mirroring alpine for this Dockerfile (and just use `alpine` or `alpine:3.14`)